app/core/exceptions.py

```
@@ -0,0 +1,641 @@
+    Base exception class for all Mobius platform exceptions.
+    
+    This provides a consistent interface for all custom exceptions
+    with proper error codes, messages, and additional context.
+    """
+    
+    def __init__(
+        self,
+        message: str,
+        error_code: ErrorCode = ErrorCode.INTERNAL_ERROR,
+        status_code: int = HTTPStatus.INTERNAL_SERVER_ERROR,
+        details: Optional[Dict[str, Any]] = None,
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)